src/components/member-profile/contribution-type/contributions-type.module.scss

```diff
@@ -42,3 +42,27 @@
     margin: 10px;
   }
 }
+
+.loadButton {
+  align-items: center;
+  background-color: white;
+  border-radius: 5px;
+  color: black;
+  cursor: pointer;
+  padding: 8px 8px;
+  font-size: 16px;
+  white-space: nowrap;
+  text-decoration: none;
+}
+
+.loadButton:hover {
+  color: white;
+  background-color: #041484;
+  transition: background-color 0.2s ease-in;
```

Comment:
it would be great if we make sure we add ease-out transition to normal class as well of loadButton else if we unhover the style will look snappy

Reply:
we fix this in the next commit.